gossipd/test/run-find_route.c

```diff
@@ -189,6 +189,7 @@ int main(void)
 	add_connection(rstate, &a, &b, 1, 1, 1);
 
 	route = find_route(tmpctx, rstate, &a, &b, 1000, riskfactor, 0.0, NULL, &fee);
+	assert(route);
```

Comment:
`tal_count(NULL)` returns 0, so I believe `assert(tal_count(route) == 1)` implicitly covers `assert(route)`.  Also applies for succeeding changes.

Reply:
Yes, you're right! I'm afraid the static analyzer needs some guidance to understand that `assert(tal_count(route) == 1)` implies `assert(route)` :-)